docker-compose.yml

```
@@ -0,0 +1,68 @@
+version: '2.4'
+
+services:
+  app:
+    image: django-geomultiplechoice
+    container_name: django-geomultiplechoice
+    restart: always
+    build: .
+    # Allow container to be attached to, e.g., to access the pdb shell
+    stdin_open: true
+    tty: true
+    ports:
+      # Map ports on your computer to ports on your container. This allows you,
+      # e.g., to visit your containerized application in a browser on your
+      # computer.
+      - 8000:8000
+    depends_on:
+      postgres:
+        condition: service_healthy
+    volumes:
+      # Mount the development directory as a volume into the container, so
+      # Docker automatically recognizes your changes.
+      - .:/app # {% if cookiecutter.local_settings != 'None' %}
+      # - ${PWD}/{{cookiecutter.local_settings}}:/app/{{cookiecutter.local_settings|replace(".example", "")}}{% endif %}
```

What's your thinking here? If we mounted this local settings file as cookiecutter is prompting, we could get rid of the local development setup step that requires devs to copy the local settings file over.

I'm also curious why this is still formatted in the cookiecutter template language, did the cookiecutter code neglect to fill this part out?

---

True! Great point. I'll clean this up. I just copied this over from cookiecutter and this can be cut.

---

Rather than cut it I think we can do something like this to automatically mount the example settings file:

```suggestion
      - ${PWD}/example/local_settings.example.py:/app/example/local_settings.py
```

---

Yes ty! I just meant cut the cookiecutter conditionals

---

Gotcha 👍